tests/Unit/Domain/Test_SizeOfElement.cpp

```diff
@@ -0,0 +1,72 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/Affine.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/DiscreteRotation.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/CoordinateMaps/Rotation.hpp"
+#include "Domain/Direction.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Mesh.hpp"
+#include "Domain/OrientationMap.hpp"
+#include "Domain/SizeOfElement.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.SizeOfElement", "[Domain][Unit]") {
```

Comment:
Could you test the tag works by setting up a DataBox and calling the compute item?